src/doc/unstable-book/src/conservative-impl-trait.md

```diff
@@ -6,5 +6,19 @@ The tracking issue for this feature is: [#34511]
 
 ------------------------
 
+The `conservative_impl_trait` feature allows a conservative form of abstract return types.
```

Comment:
Latest force push expands this section. Most of the copy is from the associated RFC.